packages/chakra-ui/src/ChakraFrameProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { CacheProvider } from "@emotion/react";
+import createCache from "@emotion/cache";
+import weakMemoize from "@emotion/weak-memoize";
+import { ChakraProvider } from "@chakra-ui/react";
+
+/**
+ * __createChakraFrameProvider is used to ensure that <Global> emotion components
+ * can be rendered within an iframe without changing the styles of the parent page.
+ * Required for using Chakra UI in the playground.
+ * 
+ * We have to define ChakraFrameProvider in this library, as opposed to the playground,
+ * in order to avoid conflicting versions of emotion, which breaks the styling.
+ * 
+ * NOTE: This is an internal component only used by @rjsf/playground (no
+ * backwards compatibility guarantees!)
+ * 
+ * From: https://codesandbox.io/s/p98y9o7jz0?file=/src/frame-provider.js:0-650
+ * Also see: https://github.com/emotion-js/emotion/issues/760#issuecomment-404353706
+ */
+
+let memoizedCreateCacheWithContainer = weakMemoize((container: HTMLElement) => {
+  let newCache = createCache({ container, key: "rjsf" });
+  return newCache;
+});
+
+export const __createChakraFrameProvider = (props: any) =>
```

Comment:
I had to do this in order to properly render Chakra UI styles into the form `iframe` in the playground, so that the styles are injected into the iframe instead of the parent document. Let me know if you know of a better / simpler way to do this, though.

Reply:
looks like a current hack we can employ at the moment, I'd let you know if I find something more straightforward, with a probably nicer and healthy-looking implementation 😆